ui/src-tauri/Cargo.toml

```diff
@@ -35,9 +35,10 @@ custom-protocol = ["tauri/custom-protocol"]
 unsafe-disable-cors = ["manta-signer/unsafe-disable-cors"]
 
 [dependencies]
-manta-signer = { path = "../../", default-features = false }
-tauri = { version = "1.0.2", default-features = false, features = ["reqwest-client", "system-tray", "updater", "window-hide", "window-show", "wry"] }
+manta-accounting = { git = "https://github.com/manta-network/manta-rs", branch = "signer_redesign", default-features = false, features = ["cocoon-fs"] }
+manta-pay = { git = "https://github.com/manta-network/manta-rs", branch = "signer_redesign", default-features = false, features = ["bs58", "groth16", "serde", "wallet"] }
```

Comment:
to keep this file as simple as possible, please re-export these crates in `manta-signer` if you need them here instead of adding them as dependencies